hoot-core/src/main/cpp/hoot/core/visitors/RemoveInvalidMultilineStringMembersVisitor.cpp

```diff
@@ -94,10 +94,20 @@ void RemoveInvalidMultilineStringMembersVisitor::visit(const ElementPtr &e)
                   rev->addElement(MetadataTags::RoleReviewee(), id2);
               }
             }
-            rev->removeElement(r->getElementId());
+            //  There is a one-to-one relation here, replace the multilinestring with the only way
+            if (rev->getMembers().size() == 1 && r->getMembers().size() == 1)
+              rev->addElement(MetadataTags::RoleReviewee(), r->getMembers()[0].getElementId());
+            //  Don't remove multilinestring relations that are members of a review relation
+            //  only contains one member that is the original multilinestring
```

Comment:
In the Port au Prince data that you sent, the original data contains no relations so these must be artifacts of hoot's conflation bookkeeping.  The following tag indicates as much:
`hoot:review:note => Missing match pair`